scripts/Setup-DevEnv.ps1

```diff
@@ -8,11 +8,11 @@ choco install git --version 2.38.1 -y
 choco install visualstudio2022community --version 117.4.2.0 -y
 choco install visualstudio2022-workload-nativedesktop --version 1.0.0 -y
 choco install visualstudio2022buildtools --version 117.4.2.0 -y
+vs_installer.exe modify --installpath "C:\Program Files\Microsoft Visual Studio\2022\Community" --add Microsoft.VisualStudio.Component.VC.Llvm.ClangToolset --quiet
```

Comment:
This should not assume that `%ProgramFiles%` is on the C drive.

Reply:
Good catch. Let me see if I can apply the same fix as I did to the yaml.